app/models/manageiq/providers/base_manager/refresher.rb

```diff
@@ -43,7 +43,7 @@ def refresh
           rescue => e
             raise if EmsRefresh.debug_failures
 
-            _log.error("#{log_ems_target} Refresh failed")
```

Comment:
Wait, @agrare, am I misreading this?

`log_backtrace` does:

`send(level, "[#{err.class.name}]: #{err.message}  Method:[#{method_name}]")`

I would have thought the message is what you want, right?  Am I going crazy?  😕 

https://github.com/ManageIQ/manageiq-gems-pending/blob/ea1f981d4c4b041264e5849185d07e9766816164/lib/gems/pending/util/vmdb-logger.rb#L103

Reply:
@jrafanie so the log line we saw was:
`ERROR -- : [Aws::EC2::Errors::Http500Error]:   Method:[block in method_missing]`

So...e.message must have been nil and e.to_s is returning something else?

Reply:
I'm guessing empty error message but I can't see where aws-sdk raises that

Reply:
It was a [`DynamicError`](https://github.com/aws/aws-sdk-ruby/blob/master/gems/aws-sdk-core/lib/aws-sdk-core/errors.rb#L181-L244) but yeah I don't see where they're setting the message or the to_s

@djberg96 do you have what the log line looked like after applying this change?  You mentioned you saw connection reset by peer.

Reply:
Bump.  @djberg96 @agrare thoughts on this?  It looks like `log_backtrace` already prints the exception message unless I'm misreading it.  I don't want to accidentally ADD code we don't need.  😉